tests/modelbench_tests/test_benchmark_runner.py

```diff
@@ -73,7 +74,10 @@ def load_plugins(self):
 
     @pytest.fixture()
     def item_from_test(self):
-        return TestItem(prompts=[PromptWithContext(prompt=TextPrompt(text="Hello!"), source_id="hello")])
+        return self.make_test_item()
+
+    def make_test_item(self, text="Hello!", source_Id="hello"):
```

Comment:
Is the capital I in `source_Id` intentional?

Reply:
It is not. Thanks for catching that!